src/utils/polygon.h

```diff
@@ -546,9 +546,16 @@ class Polygon : public PolygonRef
     {
     }
 
-    Polygon& operator=(const ConstPolygonRef& other)
+    Polygon& operator=(const ConstPolygonRef& other) = delete; // copying a single polygon is generally not what you want
+//     {
+//         path = other.path;
+//         poly = *other.path;
+//         return *this;
+//     }
```

Comment:
Also delete the implementation then. It shouldn't have been in the header anyway!

Reply:
d785d88756a48edddedc16402f6eadcfa1b9b36d